src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -154,53 +157,67 @@ public void initSpeedMode()
         rightFrontMotor.config_kI(1, SPEED_I_CONSTANT, TIMEOUT_MS);
         rightFrontMotor.config_kD(1, SPEED_D_CONSTANT, TIMEOUT_MS);
         rightFrontMotor.config_kF(1, SPEED_F_CONSTANT, TIMEOUT_MS);
-        
-	}
+
+    }
+
     public void setControlMode(ControlMode mode) {
- 		leftFrontMotor.set(mode, 0);
+        leftFrontMotor.set(mode, 0);
         rightFrontMotor.set(mode, 0);
     }
-	 public ControlMode getControlMode() {
+
+    public ControlMode getControlMode() {
         return leftFrontMotor.getControlMode();
     }
-   
-	 public void setPercentSpeedPID (double setSpeedL, double setSpeedR)
-    {
+
+    public void setPercentSpeedPID(double setSpeedL, double setSpeedR) {
         setSpeedR = MathUtil.clamp(setSpeedR, -1, 1);
-		    setSpeedL = MathUtil.clamp(setSpeedL, -1, 1);
+        setSpeedL = MathUtil.clamp(setSpeedL, -1, 1);
         leftFrontMotor.set(ControlMode.Velocity, MAX_TICKS_PER_SECOND * setSpeedL);
         rightFrontMotor.set(ControlMode.Velocity, MAX_TICKS_PER_SECOND * setSpeedR);
     }
-       
-	public boolean getReversed(){
+
+    public boolean getReversed() {
         return isReversed;
     }
-    public void setReversed(boolean r){
+
+    public boolean getMultiplierSpeed() {
+        return speedMultiplierActive;
+    }
+
+    public void setReversed(boolean r) {
         isReversed = r;
     }
-	 public void writePIDs(double output){
-		leftFrontMotor.pidWrite(output);
-		rightFrontMotor.pidWrite(-output);
- }
+
+    public void setMultiplierSpeed(boolean active, double multiplier) {
```

Comment:
So the method's named `setMultiplierSpeed`, but the internal variable is named `speedMult`. I'd say change the name of the method to `setSpeedMultiplier`